quadrat/block-templates/index.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:spacer {"height":10} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer site-header"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:template-part {"area":"header"} /-->
```

Comment:
This file is temporary, and only relevant for this demo — it sets up a template for testing the patterns.